coulomb/src/main/scala/coulomb/infra/canonical.scala

```diff
@@ -47,21 +47,7 @@ object NoImplicit {
 }
 
 object TypeString {
-  import scala.reflect.runtime.universe._
```

Comment:
The change looks like an oversimplification but there isn't that much info you can get from a `Typeable[T]`